charts/keda-operator/templates/apps_v1_deployment_operator-controller-manager.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: operator-controller-manager
```

Comment:
Maybe we could wait with optimization until we have a successful integration

Reply:
IMHO, this is not an optimization. This will be confusing once we have more than one component running. You literally can't tell which component is this except by checking the `manager` pod image name.